ext/curl/interface.c

```diff
@@ -1169,6 +1220,52 @@ PHP_MINIT_FUNCTION(curl)
 	REGISTER_CURL_CONSTANT(CURLOPT_PROXY_SSLKEY_BLOB);
 	REGISTER_CURL_CONSTANT(CURLOPT_SSLCERT_BLOB);
 	REGISTER_CURL_CONSTANT(CURLOPT_SSLKEY_BLOB);
+
+	REGISTER_CURL_CONSTANT(CURLPROTO_MQTT);
+
+	REGISTER_CURL_CONSTANT(CURLSSLOPT_NATIVE_CA);
+#endif
+
+#if LIBCURL_VERSION_NUM >= 0x074800 /* Available since 7.72.0 */
+	REGISTER_CURL_CONSTANT(CURL_VERSION_UNICODE);
+	REGISTER_CURL_CONSTANT(CURL_VERSION_ZSTD);
+#endif
+
+#if LIBCURL_VERSION_NUM >= 0x074900 /* Available since 7.73.0 */
+	REGISTER_CURL_CONSTANT(CURLOPT_SSL_EC_CURVES);
+	REGISTER_CURL_CONSTANT(CURLINFO_PROXY_ERROR);
```

Comment:
I dunno if we should map the error codes and return code.

Reply:
I sometime try to filter if they make sense or not to expose. If they are easily usable etc... For `CURLINFO_PROXY_ERROR` you're right that to make it more usable we should map all `CURLPX_*`constants. For the responses, the good answer is we should map them all. But there is so much of them... Most of the time people get the code and then call the `*_strerror()` function. But if we want to be 100% accurate we should map them all. Should I do a pass and add all the missing one ?

Reply:
If that is alright with you `CURLE_PROXY` and the `CURLPX_` serie indeed.

Reply:
Done